react-client/.expo-shared/assets.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
what is this for?

Reply:
hmn it's for optimisation purpose, I think. Not really sure - these are boiler plates generated by expo itself, it's recommended to commit thee. Can read their purpose here: https://www.webtips.dev/how-to-build-mobile-apps-with-react-native#the-structure-of-a-react-native-project